sdk/core/Azure.Core/src/RequestFailedException.cs

```diff
@@ -183,7 +183,8 @@ internal static (string FormattedError, string? ErrorCode, IDictionary<string, s
             foreach (HttpHeader responseHeader in response.Headers)
             {
                 string headerValue = response.Sanitizer.SanitizeHeader(responseHeader.Name, responseHeader.Value);
-                messageBuilder.AppendLine($"{responseHeader.Name}: {headerValue}");
```

Comment:
Are these non- `LangVersion` change required for this PR?

Reply:
No, those are fixes for breaking changes